.project

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="0.0" encoding="UTF-8"?>
```

Comment:
http://stackoverflow.com/questions/13743250/meaning-of-xml-version-1-0-encoding-utf-8